squad-server/index.js

```diff
@@ -252,6 +252,48 @@ export default class SquadServer extends EventEmitter {
     await this.logParser.watch();
   }
 
+  async setupAdminList(remoteAdminLists) {
+    try {
+      const remoteAdmins = {
+        admins: {},
+        whitelist: {},
+        groups: {}
+      };
+
+      for (let idx = 0; idx < remoteAdminLists.length; idx++) {
+        const list = remoteAdminLists[idx];
+
+        const resp = await axios({
+          method: 'GET',
+          url: `${list}`
+        });
+
+        const rawData = resp.data;
+
+        const groupRgx = /(?<=Group=)(.*?):(.*)(?=\n)/g;
+        const adminRgx = /(?<=Admin=)(\d+):(\S+)(?=\s)/g;
+
+        /* eslint-disable no-unused-vars */
+        for (const [match, groupID, groupPerms] of rawData.matchAll(groupRgx)) {
+          remoteAdmins.groups[`${idx}-${groupID}`] = groupPerms.split(',');
+        }
+        for (const [match, steamID, groupID] of rawData.matchAll(adminRgx)) {
+          const perms = remoteAdmins.groups[`${idx}-${groupID}`];
+          if (!(perms.includes('reserve') && perms.length === 1)) {
+            remoteAdmins.admins[steamID] = `${idx}-${groupID}`;
+          } else {
+            remoteAdmins.whitelist[steamID] = `${idx}-${groupID}`;
+          }
+        }
+        /* eslint-enable no-unused-vars */
+      }
+      Logger.verbose('SquadServer', 3, 'RemoteAdmins:', remoteAdmins);
```

Comment:
I think we should try to keep logs to a single line if they're not an error. Might be better to log each admin as they're processed in the loop and then perhaps `x admins loaded` at the end.